src/widget_impl/lTerm_containers_impl.ml

```diff
@@ -68,6 +69,13 @@ class virtual abox rc = object(self)
     self#compute_size_request;
     self#compute_allocations;
     self#queue_draw
+
+  method clear =
+    List.iter (fun child -> child.widget#set_parent None) children;
+    children <- [];
+    self#compute_size_request;
+    self#compute_allocations;
```

Comment:
Scratch that, they are still necessary, although they could theoretically be replaced with default values there is not much to be gained here.